src/Components/Sidebar/Sidebar.jsx

```diff
@@ -17,45 +19,93 @@ const Sidebar = () => {
           <div className="sort">
             <div className="filter-by-size">
               <label>
-                <input className="radio" type="radio" name="sort" />
+                <input
+                  className="radio"
+                  type="radio"
+                  name="sort"
+                  checked={state.sortByPrice === "lowToHigh"}
+                  value={state.sortByPrice}
+                  onChange={() => dispatch({ type: "LOW_TO_HIGH" })}
+                />
                 <span>Price- Low to High</span>
               </label>
               <label>
-                <input className="radio" type="radio" name="sort" />
+                <input
+                  className="radio"
+                  type="radio"
+                  name="sort"
+                  value={state.sortByPrice}
+                  checked={state.sortByPrice === "highToLow"}
+                  onChange={() => dispatch({ type: "HIGH_TO_LOW" })}
+                />
                 <span>Price- High to Low</span>
               </label>
             </div>
           </div>
-          <h4>Filter by price</h4>
-          <div className="price-range-star">
-            <p> 1 ⭐</p>
-            <p> 2 ⭐</p>
-            <p> 3 ⭐</p>
-            <p> 4 ⭐</p>
-            <p> 5 ⭐</p>
-          </div>
-          <div className="price-range">
-            <input className="input" type="range" min="1" max="5" step="1" />
-            <div className="range value"></div>
-          </div>
 
           <h4>Filter by Brands</h4>
           <div className="size">
             <div className="filter-by-size">
               <label>
-                <input type="checkbox" name="category-filter" />
+                <input
+                  type="checkbox"
+                  name="category-filter"
+                  checked={state.category.includes("addidas")}
+                  onChange={() =>
+                    dispatch({ type: "CATEGORY", payload: "addidas" })
+                  }
+                />
                 <span>Adidas</span>
               </label>
               <label>
-                <input type="checkbox" name="category-filter" />
+                <input
+                  type="checkbox"
+                  name="category-filter"
+                  checked={state.category.includes("air-jordan")}
+                  onChange={() =>
+                    dispatch({ type: "CATEGORY", payload: "air-jordan" })
+                  }
```

Comment:
you can just create an array of categories  and map over it instead of repeating  code

Reply:
Yes .. that's a great idea. will do that while refactoring.